store/store.go

```diff
@@ -1391,3 +1409,12 @@ func (s *Store) PaymentMethods(user *auth.UserState) (*PaymentInformation, error
 		return nil, fmt.Errorf("cannot get payment methods: unexpected HTTP code %d%s", resp.StatusCode, details)
 	}
 }
+
+func (s *Store) listRefreshFields() []string {
+	fields := append([]string{}, s.detailFields...)
+	if os.Getenv("SNAPPY_DELTA_DOWNLOAD_FORMATS") != "" {
```

Comment:
also can you make the env key a const, so we don't have to repeat it everywhere? Use `export_test.go` to keep the const private but still refer to it in tests.